client/index.js

```diff
@@ -72,6 +66,8 @@ const transactWFS = function (mode, features) {
       case 'delete':
           node = formatWFS.writeTransaction(null, null, features, formatGML);
           break;
+      default:
+        throw 'WFS-T mode ' + mode + ' is not supported'
```

Comment:
Not that in some Framework (like Angular), once you get a not catched error, the instance of the component is considered as broken and will be not used by Angular anymore (until you load another instance).
So, in some cases that's better to not throw an error, but write the error somewhere.
Here's that's completely egal.

Reply:
Thanks. The error is caught at the function call - or should it be done directly here?
I originally wanted to as you suggested and just log the error, but I think we also need to escape the function in this case. How would you do it? As I understand we should not use a return if we don't use it in other cases, is that right?

Reply:
No it's ok, you do a try catch. That was just to warn you about throwing error in JS.
Otherwise, you can add no default and add just after the switch-case:
```
console.log('my error')
return;
```
To exit the function whitout throwing errors

Reply:
Great, thanks for the comments